app/lobby/server/lobby.js

```diff
@@ -3,6 +3,10 @@ Meteor.methods({
     let game_proposer = new GameProposer(player_ids, exclusions)
     game_proposer.propose()
   },
+  proposeGameFromKingdom: function(player_ids, exclusions, kingdom_id) {
```

Comment:
If we consolidate the two proposals to the same Meteor call, then we just need to modify the original `proposeGame` call to this:

```
proposeGame: function(player_ids, exclusions, kingdom_id) {
  let game_proposer = new GameProposer(player_ids, exclusions, kingdom_id)
  game_proposer.propose()
},
```